react/features/welcome/components/BlankPage.native.js

```diff
@@ -54,7 +53,7 @@ class BlankPage extends Component<*> {
      */
     render() {
         return (
-            <LocalVideoTrackUnderlay style = { styles.blankPage }>
+            <LocalVideoTrackUnderlay>
```

Comment:
Do we need a StatusBar component here?

Reply:
Hmm, not sure what you mean. I'd say not, but is there a header there?